### PR TITLE
Automate clickhouse migration

### PR DESCRIPTION
**Pull request by LucoLe**
#### Summary

Adds a mix task that generates an empty clickhouse migration.

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/mix/tasks/gen_clickhouse_migration.ex [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  def run(args) do
+    [name] = args
+    file = "db/#{timestamp()}_#{underscore(name)}.sql"
+    create_file(file, "")
+  end
+
+  defp timestamp do
+    {{y, m, d}, {hh, mm, ss}} = :calendar.universal_time()
+    "#{y}#{pad(m)}#{pad(d)}#{pad(hh)}#{pad(mm)}#{pad(ss)}"
```
Comment (IvanIvanoff) [inline lib/mix/tasks/gen_clickhouse_migration.ex]: Use https://hexdocs.pm/timex/Timex.Format.DateTime.Formatter.html#c:format/2

**Comment by IvanIvanoff** [comment]
1. You won't need this task. All you need to do is `mix ecto.gen.migration -r Sanbase.ClickhouseRepo example_migration`

2. In order to do CH migrations you need to have the repo in `ecto_repos` otherwise the migrations won't be run at all. Also have in mind that the CH structure is not managed by sanbase, so the setup should be similar to Timescale - these migrations must not run in prod.

**Comment by LucoLe** [comment]
I'm confused is this task still relevant or not? If it's relevant I don't understand what should be done because in the description sits something completely different.

**Comment by IvanIvanoff** [comment]
@LucoLe https://github.com/santiment/clickhouse-tables is managing the ClickHouse database. There we roll out an in-house migration system that is lacking this functionality. It should be implemented there as a bash/python script.

**Pull request closed without merging**